lib/pavilion/test_config/setup.py

```diff
@@ -349,21 +349,32 @@ def resolve_inheritance(base_config, suite_cfg, suite_path):
     # A list of tests whose parent's have had their dependencies
     # resolved.
     ready_to_resolve = list()
-    for test_cfg_name, test_cfg in suite_cfg.items():
-        if test_cfg.get('inherits_from') is None:
-            test_cfg['inherits_from'] = '__base__'
-            # Tests that depend on nothing are ready to resolve.
-            ready_to_resolve.append(test_cfg_name)
-        else:
-            depended_on_by[test_cfg['inherits_from']].append(test_cfg_name)
-
-        try:
-            suite_tests[test_cfg_name] = TestConfigLoader().normalize(test_cfg)
-        except (TypeError, KeyError, ValueError) as err:
-            raise TestConfigError(
-                "Test {} in suite {} has an error: {}"
-                .format(test_cfg_name, suite_path, err))
+    if suite_cfg is None: # Catch null test suites.
+        raise TestConfigError("Test Suite {} is empty.".format(suite_path))
+    try:
+        for test_cfg_name, test_cfg in suite_cfg.items():
+            if test_cfg is None:
```

Comment:
I think this is actually a bug in yaml_config, but this is all a reasonable fix for now.